source/docs/software/dashboards/smartdashboard/test-mode-and-live-window/enabling-test-mode.rst

```diff
@@ -1,15 +1,31 @@
 Enabling Test mode (LiveWindow)
 ===============================
 
-You may add code to your program to display values for your sensors and actuators while the robot is in Test mode. This can be selected from the Driver Station whenever the robot is not on the field. The code to display these values is automatically generated by RobotBuilder and is described in the next article. Test mode is designed to verify the correct operation of the sensors and actuators on a robot. In addition it can be used for obtaining setpoints from sensors such as potentiometers and for tuning PID loops in your code.
+You may add code to your program to display values for your sensors and actuators while the robot is in Test mode. This can be selected from the Driver Station whenever the robot is not on the field (see :ref:`docs/software/basic-programming/using-test-mode:Enabling Test Mode` for details on how to do this). Test mode is designed to verify the correct operation of the sensors and actuators on a robot. In addition it can be used for obtaining setpoints from sensors such as potentiometers and for tuning PID loops in your code.  When the robot is enabled in Test mode, the SmartDashboard display will switch to test mode (LiveWindow) and will display the status of any actuators and sensors used by your program.
 
-Setting Test mode with the Driver Station
------------------------------------------
 
-.. image:: images/enabling-test-mode/setting-test-mode-driver-station.png
-   :alt: Selecting the "Test" button on the Driver Station and then "Enable".
+.. important:: Since 2024, LiveWindow is not enabled by default in Test mode!
+
+Enabling LiveWindow in Test Mode
+--------------------------------
+
+To run LiveWindow in Test Mode, the following code is needed in the ``Robot`` class:
+
+.. tab-set-code::
+
+    .. code-block:: java
+
+        @Override
+        public void robotInit() {
+          enableLiveWindowInTest(true);
+        }
+
+    .. code-block:: cpp
+
+        void Robot::RobotInit() {
+          EnableLiveWindowInTest(true);
+        }
 
-Enable Test Mode in the Driver Station by clicking on the "Test" button and setting "Enable" on the robot. When doing this, the SmartDashboard display will switch to test mode (LiveWindow) and will display the status of any actuators and sensors used by your program.
```

Comment:
Isn't this still needed?

Reply:
It's a duplicate section, replaced with a link in the previous section.